modules/about/con_understanding-internal-developer-platforms.adoc

```diff
@@ -0,0 +1,72 @@
+* *Central configuration:* Infrastructure and policies are defined as code and centrally managed, reducing drift and maintenance overhead.
+* *Governance at scale:* Policies and best practices are embedded into developer workflows using automation and templates, without adding friction to the process.
+
+For organizations::
+
+* *Scalability:* {product-very-short} enables consistent developer onboarding and application delivery across growing teams and environments.
+* *Security:* Role-based access control (RBAC) and integration with enterprise systems ensure access is managed securely and in line with compliance requirements.
+* *Operational efficiency:* By removing manual handoffs and centralizing key development workflows, {product-very-short} improves time to value and increases return on engineering investment.
+
+== Key features
+
+Centralized dashboard::
+Access development tools, CI/CD pipelines, APIs, monitoring tools, and documentation from a single interface. Integrate with systems like Git, {ocp-brandname}, Kubernetes, and JIRA.
+
+Configuration management::
+Manage updates and configurations through centralized repositories, improving version control and consistency across environments.
+
+Learning paths::
+Guide developers through structured tutorials and onboarding steps. Help teams upskill with internal and Red Hat training resources in one place. For more information, see []
+
+Plugins and integrations::
+Extend {product-very-short} with verified plugins that add new functionality without downtime. Dynamically integrate with supported tools such as Tekton for pipelines, GitOps for deployment automation, Nexus Repository for artifact storage, and JFrog Artifactory. {product-very-short} also supports connecting to {ocp-brandname}, CI/CD systems, and security scanner through Red Hat-curated extensions.
+
+Role-Based Access Control (RBAC)::
+Manage user access with robust security permissions tailored to organizational needs.
+
+Software catalog::
+Search, view, and manage services, APIs, and libraries from a central inventory. Track ownership, metadata, and component health in one place.
+
+Software templates::
+Accelerate project setup using preconfigured templates for CI/CD, runtime, and security. Standardize implementation while enabling developer autonomy.
+
+Tech docs::
+Create, store, and view technical documentation alongside code. Make content searchable, consistently formatted, and accessible through the portal.
+
+Scalability::
+Support growing teams and applications while maintaining access to the same tools and services.
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about software templates, see link:https://docs.redhat.com/documentation/red_hat_developer_hub/{product-version}/html/customizing_red_hat_developer_hub/configuring-templates[Configuring templates].
```

Comment:
Consider removing the leading sentence in the additional resources section. Perhaps we could just add the text embedded with links, similar to the format most CCS follow.
```suggestion
* link:https://docs.redhat.com/documentation/red_hat_developer_hub/{product-version}/html/customizing_red_hat_developer_hub/configuring-templates[Configuring templates].
```